subprojects/packagefiles/sdl2/src/joystick/meson.build

```diff
@@ -13,6 +13,7 @@ elif host_machine.system() == 'windows'
   subdir('windows')
 elif host_machine.system() == 'darwin'
   subdir('darwin')
+  subdir('iphoneos')
```

Comment:
despite being called `iphoneos` we still need it because it defines `SDL_IOS_JoystickDriver` which is used across MacOS and iOS